src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Utilities.cs

```diff
@@ -60,6 +60,21 @@ public static IEnumerable<string> GetTarballContentNames(string tarballPath)
         }
     }
 
+    public static void ExtractNupkg(string package, string outputDir)
+    {
+        Directory.CreateDirectory(outputDir);
+
+        using (ZipArchive zip = ZipFile.OpenRead(package))
+        {
+            foreach (ZipArchiveEntry entry in zip.Entries)
+            {
+                string outputPath = Path.Combine(outputDir, entry.FullName);
+                Directory.CreateDirectory(Path.GetDirectoryName(outputPath));
+                entry.ExtractToFile(outputPath);
+            }
+        }
```

Comment:
You can remove some extra lines and indentation with this pattern.

```suggestion
        using ZipArchive zip = ZipFile.OpenRead(package);
        foreach (ZipArchiveEntry entry in zip.Entries)
        {
            string outputPath = Path.Combine(outputDir, entry.FullName);
            Directory.CreateDirectory(Path.GetDirectoryName(outputPath));
            entry.ExtractToFile(outputPath);
        }
```